server/src/db/migrations/20211102194503_machines_table.ts

```diff
@@ -3,37 +3,38 @@ import { Knex } from "knex";
 
 export async function up(knex: Knex): Promise<void> {
     
-    return knex.schema.hasTable('MachineFamilies').then(function(exists) {
+    return knex.schema.hasTable('EquipmentLabels').then(function(exists) {
         if (!exists) {
-            return knex.schema.createTable('MachineFamilies', function(t) {
+            return knex.schema.createTable('EquipmentLabels', function(t) {
                 t.increments('id');
                 t.string('name', 50);
-                t.text('description');
-                t.integer('training_module').references('id').inTable('TrainingModule');
+                t.specificType('trainingModules', 'integer ARRAY')
```

Comment:
Doesn't this fail 1st normal form? I think its a lot better practice to have a separate table to represent the multiple values